packages/react-ui-core/src/Gmap/withGoogleScript.js

```diff
@@ -33,20 +36,31 @@ export default function(BaseComponent) {
     }
 
     state = {
-      loaded: false,
+      loaded: this.props.loaded || false,
       spinner: this.props.spinner,
     }
 
     componentDidMount() {
-      if (this.isInitialized) {
-        this.scriptLoaded()
+      if (this.state.loaded || (this.hasExistingScript && this.isInitialized)) {
+        // no other place to put this due to SSR
+        this.setState({ loaded: true }) // eslint-disable-line react/no-did-mount-set-state
         return
```

Comment:
is the return necessary?

Reply:
Yes, because we don't need any of the following to be called if the api has fully loaded.